eventing/samples/github-events/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM golang AS builder
+
+WORKDIR /go/src/github.com/knative/docs/
+ADD . /go/src/github.com/knative/docs/
```

Comment:
It looks like you don't need to use `dep`, so this is much cleaner than what I had to do in the [`k8sevents` commit](https://github.com/knative/docs/pull/240) (see https://github.com/knative/docs/pull/240/commits/f00e99d5af455ce8add415175d3f450fb8631428 and https://github.com/knative/docs/pull/240/commits/d1b685ad65a672a6eacc421ce90b52c7b389b86e for the `dep` changes). Prior to needing `dep`, I used the following, which is even simpler than what you have here, and works from the sample directory:

```docker
FROM golang AS builder

ADD . /go/src/k8s-events
# Build the event processing function command inside the container.
# (You may fetch or manage dependencies here,
# either manually or with a tool like "godep".)
RUN go get k8s.io/api/core/v1 github.com/knative/eventing/pkg/event
RUN go install k8s-events

FROM gcr.io/distroless/base
COPY --from=builder /go/src/github.com/knative/docs/k8s-events /sample
...
```

Reply:
I copied the k8s event docker file and changed the path of the runable. I was able to get this to work testing the README script just fine.